pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole.go

```diff
@@ -312,3 +313,34 @@ func newMobileClientAdminRoleBinding(cr *mdcv1alpha1.MobileDeveloperConsole) (*r
 		},
 	}, nil
 }
+
+func newMobileClientAdminRole(cr *mdcv1alpha1.MobileDeveloperConsole) (*rbacv1.Role, error) {
+	return &rbacv1.Role{
```

Comment:
Note that this version of the role doesn't provide permissions to the CRs from the metrics service. If we ever re-add that service into the mix we can add whatever is needed here again.